packages/cfd/src/features/Containers/cfd-password-modal/icon-type.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Icon } from '@deriv/components';
+import { CFD_PLATFORMS, routes } from '@deriv/shared';
+import TradingPlatformIcon from '../../../Assets/svgs/trading-platform';
+
+type TIconTypeProps = {
+    platform: string;
+    type?: string;
```

Comment:
Proper typing pls

Reply:
We can not use it now because we are still using store and it has messed up types.
it can be properly typed after everything is coming from api. 😵‍💫